src/errors.ts

```diff
@@ -0,0 +1,31 @@
+interface ErrorExtensions {
+  properties?: Record<string, unknown>;
+  statusCode?: string;
+  response?: Record<string, unknown>;
+  kind?: string;
+}
+
+export interface OneSdkError extends ErrorExtensions, Error {}
+
+interface WrappedError extends Error {
+  originalError?: unknown;
+  extensions?: ErrorExtensions;
+}
+
+export function isOneSdkError(err: unknown): err is OneSdkError {
+  return typeof err === 'object' && err != null && 'message' in err;
```

Comment:
suggestion: I am not very aligned with this check, because any error will be considered OneSdkError.

I checked OneSDK errors implementation
https://github.com/superfaceai/one-sdk-js/blob/v1.5.2/src/internal/errors.ts
https://github.com/superfaceai/one-sdk-js/blob/v1.5.2/src/internal/interpreter/profile-parameter-validator.errors.ts
https://github.com/superfaceai/one-sdk-js/blob/v1.5.2/src/internal/interpreter/map-interpreter.errors.ts

and it looks that with weird exception of `SDKExecutionError` all extend `ErrorBase`. 

```ts
import {
  ErrorBase,
  SDKExecutionError,
  SDKBindError,
} from '@superfaceai/one-sdk';

interface ErrorExtensions {
  properties?: Record<string, unknown>;
  statusCode?: string;
  response?: Record<string, unknown>;
  kind?: string;
}

export interface OneSdkError extends ErrorExtensions, Error {}

export function isOneSdkError(err: unknown): err is OneSdkError {
  return typeof err === 'object' && err != null && 'message' in err;
}

// -------------------------------------------

const plainErr = new Error('Plain error');
const baseErr = new ErrorBase('MyKind', 'Error base');
const execErr = new SDKExecutionError(
  'Execution error',
  ['execution error first line', 'execution error second line'],
  ['hint 1', 'hint 2']
);
const bindErr = new SDKBindError(
  'Bind error',
  ['bind error first line', 'bind error second line'],
  ['hint 1', 'hint 2']
);

console.log('plainErr instanceof Error', plainErr instanceof Error);
console.log('baseErr instanceof ErrorBase', baseErr instanceof ErrorBase);
console.log(
  'execErr instanceof SDKExecutionError',
  execErr instanceof SDKExecutionError
);
console.log('bindErr instanceof SDKBindError', bindErr instanceof SDKBindError);
```

And I think here we should be mapping only [MapInterpreterErrorBase](https://github.com/superfaceai/one-sdk-js/blob/v1.5.2/src/internal/interpreter/map-interpreter.errors.ts#L10).

Other errors, such as bind error, or profile validation would get use of custom error with user understable description.

So I believe that `isOneSdkError` should be doing `err instanceof ErrorBase` and `error instanceof SDKExecutionError`. Second needs to be cleared by folks why it doesn't inherint from ErrorBase.

Reply:
I have adjusted the `isOneSdkError` as you suggested. However, I kept `OneSdkError` interface as it is for now, since I prefer more explicit control over what properties are being used. I propose revising it later if there are changes in errors hierarchy.